src/ServicePulse.Host/app/js/views/message/editor/messageEditorModal.html

```diff
@@ -7,6 +7,9 @@ <h3 class="modal-title">Edit and retry message</h3>
             other subscribers, editing it now has the risk of changing the semantic meaning of the event and may result in
             altering the system behavior.</div>
     </div>
+    <div class="row alert alert-warning" ng-show="!message.isContentTypeSupported">
+        <div class="col-sm-12"><i class="fa fa-exclamation-circle"></i> Message body cannot be edited: content type ({{message.bodyContentType}}) not supported. Only msessages with body content serizalised as Json or XML can be edited.</div>
```

Comment:
Fix typos and wording:

`<div class="col-sm-12"><i class="fa fa-exclamation-circle"></i> Message body cannot be edited because content type ({{message.bodyContentType}}) is not supported. Only messages with body content serialized as JSON or XML can be edited.</div>`